Lib/test/test_sys.py

```diff
@@ -1210,6 +1210,13 @@ def test_pystats(self):
         sys._stats_clear()
         sys._stats_dump()
 
+    @test.support.cpython_only
+    @unittest.skipUnless(hasattr(sys, 'abiflags'), 'need sys.abiflags')
```

Comment:
I don't understand this skip. When sys.abiflags is not available? Nothing says in the doc that it's optional: https://docs.python.org/dev/library/sys.html#sys.abiflags

Reply:
It's only available on POSIX systems (not Windows).

Reply:
Oh, I didn't know. Would you mind to add ``   .. availability:: Unix.`` to https://docs.python.org/dev/library/sys.html#sys.abiflags ?

Reply:
I took the freedom to including this change as part of your PR.